apis/python/tests/test_dataframe.py

```diff
@@ -34,16 +34,6 @@ def _schema():
 
 def test_dataframe(tmp_path, arrow_schema):
     uri = tmp_path.as_posix()
-
```

Comment:
In future, also could be a separate PR. (Nice catch! But unrelated to the main purpose of this PR.)

Reply:
Done: #3769